app/views/service_providers/_form.html.slim

```diff
@@ -8,14 +8,17 @@
 = form.input :description
 = form.input :issuer
 = form.association :agency
-= form.input :acs_url, label: 'Assertion Consumer Service URL'
-= form.input :assertion_consumer_logout_service_url, label: 'Assertion Consumer Logout Service URL'
-= form.input :sp_initiated_login_url, label: 'SP Initiated Login URL'
-= form.input :block_encryption, as: :radio_buttons, label: 'Block Encryption'
-= form.input :saml_client_cert, label: 'SAML Client Cert (PEM encoded)'
-= form.input :return_to_sp_url, label: 'Return to SP URL'
+= form.input :acs_url, label: t('service_providers.labels.acs_url')
```

Comment:
these labels can be customized with simple_form

this is the default simple_form locale file: https://github.com/plataformatec/simple_form/blob/c3a307cb9501208e053f771738e34a3d93d33858/lib/generators/simple_form/templates/config/locales/simple_form.en.yml

if you copy it into `config/locales` you can add a label for this input like so (no need to set the `label` option, SimpleForm will find it for you!

```
# simple_form.en.yml

en:
  simple_form:
    labels:
      defaults:
         service_provider:
           acs_url: 'Assertion Consumer Service URL'
```

see how we did this for micropurchase here: https://github.com/18F/micropurchase/blob/f8b2f3d5c1b8c41a262e1ef2642764a395a88dc1/config/locales/defaults/simple_form.en.yml